apps/web/src/core/utils/terminal-parser.ts

```diff
@@ -0,0 +1,27 @@
+import type { Terminal } from "xterm"
```

Comment:
Same comment here re: file organization - would be great if we could move all dependencies of the vm demo into one folder and same for agents

Reply:
Done :D